ansible/roles/python-devel/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# tasks file for roles/python-devel
+
+- name: install python-devel
+  sudo: True
+  yum: name={{ item }}  state=present
+  with_items: [ "python-devel", "gcc" ]
```

Comment:
Only remaining thing is a style issue- can you use the long form (makes it easier to review diffs in future), see e.g. https://github.com/openmicroscopy/infrastructure/blob/master/ansible/roles/omero-basedeps/tasks/main.yml#L9


and `sudo`, we should be consistent (add it to all required tasks?)